.docker.env

```diff
@@ -0,0 +1,11 @@
+# NOTE TO DEVELOPERS
+# Please do not directly change the values of the variables specified in this file, as this may lead to unexpected behavior of your local development server. Specifically:
+## Additionally, changing the variables (which are flags for running the local development server) to true will permanently enable the flag to your `make run/run-offline` command even you do not explicitly pass the flag.
```

Comment:
Is there some text missing here? There's nothing after "Specifically:"

Also, to me "permanently" suggests that if a dev changes a variable to true and then back to false, the flag will stay enabled. Is that the case, or do you just mean that the flag will be enabled even if not passed on the command line?

Reply:
fixed the comments.
for the second ques --
the flag will be enabled even if it is not passed in the make command from CLI (if dev changes any variable to true), otherwise no permanent effect if they set them back to false.